ci/pipeline.yml

```diff
@@ -0,0 +1,168 @@
+---
+groups:
+  - name: pcap-release
+    jobs:
+      - unit-tests
+      - unit-tests-pr
+
+jobs:
+  - name: unit-tests
+    public: true
+    serial: true
+    plan:
+      - do:
+          - get: git
+            trigger: true
+          - task: lint
+            config:
+              platform: linux
+              image_resource:
+                type: docker-image
+                source:
+                  repository: iacbox.common.repositories.cloud.sap/haproxy-boshrelease-testflight
```

Comment:
if we aren't using any of the haproxy-specific packages but `apt-install` them in the scripts anyways, why don't we simply run the public `bosh/main-bosh-docker` image for the time being?